gprofiler/profilers/perf.py

```diff
@@ -77,10 +77,10 @@ def get_average_frame_count(samples: Iterable[str]) -> float:
         if len(kernel_split) > 1:
             # example: "a;b;c;d_[k];e_[k] 1" should return the same value as "a;b;c 1", so we don't
             # add 1 to the frames count like we do in the other branch.
-            frame_count_per_samples.append(kernel_split[0].count(";"))
+            frame_count_per_samples.append(kernel_split[0].count(";") - kernel_split[0].count("[unknown]"))
```

Comment:
1. Document what is the purpose for this `count`.
2. Now we added more logic to both branches, perhaps share it? i.e you can run both `count` calls of `;` and `unknown` before the branch, then use the same variable.